dspace-api/src/main/java/org/dspace/discovery/IndexClient.java

```diff
@@ -58,8 +58,8 @@ public static void main(String[] args) throws SQLException, IOException, SearchS
         Context context = new Context(Context.Mode.READ_ONLY);
         context.turnOffAuthorisationSystem();
 
-        String usage = "org.dspace.discovery.IndexClient [-cbhf] | [-r <handle>] | [-i <handle>] or nothing to " +
-            "update/clean an existing index.";
+        String usage = "org.dspace.discovery.IndexClient [-cbhf] | [-r <handle>] | [-i <handle>] | [-item_uuid " +
```

Comment:
Can't we simplify this to `-i <handle or uuid>`? This would prevent adding the `-item_uuid` option.

Reply:
done